packages/libs/chainwebjs/package.json

```diff
@@ -52,7 +52,6 @@
     "format": "pnpm run /^format:.*/",
     "format:md": "remark README.md -o --use @kadena-dev/markdown",
     "format:src": "prettier config src --write",
-    "preinstall": "npx only-allow pnpm",
```

Comment:
We define this in the root `package.json` already.

Reply:
Does this have an effect on clients installing chainwebjs?

Reply:
Yes, they are no longer forced to use pnpm